### PR TITLE
Add upcoming funding card to v2project page

### PR DESCRIPTION
**Pull request by johnnyd-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues,
Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, 
Screen Shot 2022-04-11 at 8 14 21 pm
provide screenshots or screen recordings to demonstrate the
changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).

### CODE REVIEW

File: src/components/v2/V2Project/V2FundingCycleSection/PayoutSplitsCard.tsx [hunk 1]
```diff
@@ -16,7 +16,11 @@ import { useETHPaymentTerminalFee } from 'hooks/v2/contractReader/ETHPaymentTerm
 
 import DistributePayoutsModal from './modals/DistributePayoutsModal'
 
-export default function PayoutSplitsCard() {
+export default function PayoutSplitsCard({
+  hideDistributeBtn,
```
Comment (aeolianeth) [inline src/components/v2/V2Project/V2FundingCycleSection/PayoutSplitsCard.tsx:20]: **nitpick** kinda prefer full words for stuff for max readability and min confusion, wdyt:

```suggestion
  hideDistributeButton,
```

File: src/components/v2/V2Project/V2FundingCycleSection/UpcomingFundingCycle.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          }
+          fundingCycleDurationSeconds={queuedFundingCycle.duration}
+          fundingCycleStartTime={queuedFundingCycle.start}
+          isFundingCycleRecurring={true}
+          fundingCycleRiskCount={V2FundingCycleRiskCount(queuedFundingCycle)}
+          expand={expandCard}
+        />
+      </CardSection>
+
+      <PayoutSplitsCard hideDistributeBtn />
```
Comment (aeolianeth) [inline src/components/v2/V2Project/V2FundingCycleSection/UpcomingFundingCycle.tsx:37]: **question**: Doesn't this need to take into account the upcoming funding cycle's payouts?

**Review by aeolianeth — CHANGES_REQUESTED** [review]
@johnnyd-eth nice, lgtm, jsut a few comments and will need to `yarn i18n:extract` to make CI happy

File: src/components/v2/V2Project/V2FundingCycleSection/UpcomingFundingCycle.tsx (hunk 2 above)
Comment (johnnyd-eth) [reply src/components/v2/V2Project/V2FundingCycleSection/UpcomingFundingCycle.tsx]: Yep lol my bad. Done now

**Comment by johnnyd-eth** [comment]
@aeolianeth addressed your latest

File: src/components/v2/V2Project/V2FundingCycleSection/UpcomingFundingCycle.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+import PayoutSplitsCard from './PayoutSplitsCard'
+import ReservedTokensSplitsCard from './ReservedTokensSplitsCard'
+
+export default function UpcomingFundingCycle({
+  expandCard,
+}: {
+  expandCard?: boolean
+}) {
+  const {
+    queuedFundingCycle,
```
Comment (aeolianeth) [inline src/components/v2/V2Project/V2FundingCycleSection/UpcomingFundingCycle.tsx:18]: this is fine for now but I think we don't need to have all of this queued stuff in project context. We can load it only when necessary to save some Infura burden.

Comment (johnnyd-eth) [reply src/components/v2/V2Project/V2FundingCycleSection/UpcomingFundingCycle.tsx:18]: Ah yeah true wasn't considering that at all. Will defs be worth removing eventually

**Review by aeolianeth — APPROVED** [review]

**Pull request merged**